### ISSUE TITLE
pytave: failures returning various empty matrices

### ISSUE THREAD

**Issue by cbm755** [cbm755/octsympy#475]
Very few failures in the `pytave` branch now!

Many (most?) of the remaining ones seem to be about empty matrices which are not 0x0 matrices: e.g., 0x2 matrices.  These seem to get squashed to `0x0`, which is causing test failures.

````
>> a = zeros(2,0)
a = [](2x0)
>> sym(a)
ans = (sym) []  (empty 0×0 matrix)
````

**Comment by cbm755** [comment]
see also the failing test in `@sym/sym`:

````
>> syms n m integer
>> A = sym('A', [n m]);
>> A
A = (sym) A  (1×1 matrix expression)
````

This is related.

**Comment by cbm755** [comment]
cc @genuinelucifer

**Comment by genuinelucifer** [comment]
I tested and yes it seems to occur..  
.   
I had one (unrelated) question:  
I paused trying to build pytave on windows. And instead shifted my attention to using 'pycall' to save variables to python. After getting so many errors I was thinking if it is good to continue with that effort or to try and build pytave on windows first?

**Comment by cbm755** [comment]
> I paused trying to build pytave on windows. And instead shifted my
> attention to using 'pycall' to save variables to python. After getting
> so many errors I was thinking if it is good to continue with that effort
> or to try and build pytave on windows first?

Let's do both ;-)

Feels like we're getting close with pytave import/export of variables!

**Comment by genuinelucifer** [comment]
> Let's do both ;-)

Sure,  it's just that there were sooo many different errors all appearing
at once with the new 'pycall' approach to save that I completely forgot
about building pytave on windows.
With your changes, it looks good now and hopefully we can complete the
pycal part soon. (atleast to the level that most of the test cases start
passing again)

> Feels like we're getting close with pytave import/export of variables!
>

Definitely. On the import side, I guess only @pyobject is left. While on
the export side there is a lot to do but probably can be done without much
effort now.

**Comment by cbm755** [comment]
> Sure, it's just that there were sooo many different errors all appearing
> at once with the new 'pycall' approach to save that I completely forgot
> about building pytave on windows.

Re: Windows: You can always try with the `pytave` branch (instead of
your `pytave_ex` branch).

**Comment by cbm755** [comment]
Oops, this should be fixed by 5b479371092cafe2291dd9100fc24d255e3ffb9.
